molecule/provisioner/ansible/playbooks/docker/create.yml

```diff
@@ -3,7 +3,7 @@
   hosts: localhost
   connection: local
   gather_facts: false
-  no_log: "{{ not lookup('env', 'MOLECULE_DEBUG') | bool }}"
+  no_log: "{{ not (lookup('env', 'MOLECULE_DEBUG') | bool or molecule_yml.provisioner.log|default(false) | bool) }}"
```

Comment:
If `provisioner.log` is set to `true` in `molecule.yml`, then I should expect `no_log` to be `false`. However, `no_log` is being evaluated to `true`  here.